html/changelog.html

```diff
@@ -51,6 +51,73 @@
 		
 		-->
 		<div class="commit sansserif">
```

Comment:
you need to turn github actions OFF on your repository to stop changelog compiles happening automatically